packages/design-system/src/components/reset/resetFormEls.stories.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+
+export default {
+  title: 'Reset / Form Elements',
+};
+
+export const FormElements = () => {
+  return (
+    <article>
+      <hgroup>
+        <h2>Form Elements</h2>
+        <p>This page showcases how reset styles impact HTML form element tags.</p>
+      </hgroup>
+
+      <article>
+        <h3>Text fields</h3>
+
+        <h4>Text inputs</h4>
+        <div className="preview">
+          <form action="">
+            <fieldset>
+              <legend>Legend: Text input examples</legend>
+              <label htmlFor="text-input">Default text input field</label>
+              <input type="text" name="text-inputs" id="text-input" />
+              <br />
+              <label htmlFor="disabled-text-input">Disabled text input field</label>
+              <input type="text" name="text-inputs" id="disabled-text-input" disabled />
+              {/* <br />
+              <label htmlFor="readonly-text-input">Readonly text input field</label>
+              <input type="text" name="text-inputs" id="readonly-text-input" readOnly /> */}
```

Comment:
`readonly` are common text inputs types, but we don't use them in our DS, so I can't infer style. Is this something we need to support?

Reply:
That's a good idea. I can't think of one of our products that uses them, but that doesn't mean they aren't or won't be used. Seems like we could do something similar to or the same as a disabled style? @line47, what do you think? [Here's an MDN page on readonly fields](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly).

Reply:
I think it's worth having a conversation with @mliupaeter about it before implementing. ALso I'm unsure if teams are using readonly fields. :shrug:

Reply:
I can omit the `readonly` example for now. We can revisit once there's a known need later on.